pkg/stores.go

```diff
@@ -0,0 +1,37 @@
+package ocr2keepers
+
+import "fmt"
+
+type ResultStore[T any] interface {
+	Add(T)
+	Remove(T)
+	View() ([]T, error)
+}
+
+type resultStore[T any] struct {
+	data map[string]T
+}
+
+func NewResultStore[T any]() *resultStore[T] {
+	return &resultStore[T]{
+		data: make(map[string]T),
+	}
+}
+
+func (s *resultStore[T]) Add(result T) {
+	key := fmt.Sprintf("%v", result)
```

Comment:
Would be necessary to introduce a key instead of encoding the object that we are saving as a key (or at least use something shorter such as sha256 or md of the data)

Reply:
The scope of this card is just to get a bare-bones result store in place as a required dependency. There is a follow-up card for applying more detail. I'll add this note there for future implementation.